test/test_sparse_csr.py

```diff
@@ -79,6 +81,8 @@ def test_factory_size_check(self, device, dtype):
 
     @onlyCPU
     def test_sparse_csr_print(self, device):
+        orig_maxDiff = self.maxDiff
+        self.maxDiff = None
```

Comment:
This test was wrong from the beginning. It uses genSparseCSRTensor which generates a random csr tensor however this test compares with an expected and static string. 
See :  https://github.com/pytorch/pytorch/blob/f4320cc526dad32f8dcc2798e9a48086c58f0698/test/test_sparse_csr.py#L223

Reply:
should we remove/skip this for now?

Reply:
I would suggest to generate csr tensor in this way: 

https://github.com/pytorch/pytorch/blob/f4320cc526dad32f8dcc2798e9a48086c58f0698/test/test_sparse_csr.py#L242

So when CUDA support is ready for merge it will not generate to much conflicts